staging/src/k8s.io/apimachinery/pkg/util/validation/validation_test.go

```diff
@@ -154,6 +154,22 @@ func TestIsValidPortNum(t *testing.T) {
 	}
 }
 
+func TestIsInRange(t *testing.T) {
+	goodValues := [][]int{{1, 0, 10}, {5, 5, 20}, {25, 10, 25}}
```

Comment:
suggest:
```
goodValues := []struct{
  value int
  min int
  max int
}{{1, 0, 10}, {5, 5, 20}, {25, 10, 25}}
```
and then you can do
```
IsInRange(val.value, val.min. val.max)
```
which reads more clearly.
Similarly for `badValues`.